doc/decisions/database_plant_hierarchy.md

```diff
@@ -85,6 +87,28 @@ Cons:
 - Almost everything in the plants table needs to be nullable.
 - More complex insert and update logic.
 
+### One table per taxonomy rank and one for concrete plants. + View and custom insert/update/delete functionality
+
+[Example](example_migrations/one-table-per-taxonomy-view-functions)
+
+It's similar to `One table for taxonomy ranks and one for concrete plants` We are extending it with a view and custom functions to reduce insert and update complexity in the backend.
+
+Pros:
+
+- Inserting new plants is easy. We only need to implement minor backend changes.
+
+Cons:
+
+- Attribute overrides can only be done on the variety or cultivar level.
+- More complex insert and update logic.
```

Comment:
But it would be hidden behind triggers? In general this sounds like a good idea, as we then don't need to keep Rust, maybe Python (E2E) and JavaScript code (scraper) in sync.

Reply:
Yes, and the view would act just like our plant table right now, we only extend it with some more properties.